villain/compile.rkt

```diff
@@ -58,6 +61,9 @@
            (Extern 'raise_error_align)
            (Extern 'str_to_symbol)
            (compile-defines ds))]))
+;           (compile-λ-definitions (λs
```

Comment:
We shouldn't need `compile-defines` once there's `compile-λ-definitions`.

Reply:
Yes, thanks!  I kept them for the libraries now.  You already have written the code for generating the list of the ids of all the functions in the libraries (`stdlib-ids`).  We can include the library functions via `(define (stdlibs program) (Letrec fs ls program)` and then `(compile (stdlibs program))`.  I will write the code for generating the corresponding lambdas so that we can do this.